web/src/app/loading/components/Spinner.tsx

```diff
@@ -51,23 +66,6 @@ export default function Spinner(props) {
   )
 }
 
-Spinner.propTypes = {
-  // Wait `delayMs` milliseconds before rendering a spinner.
-  delayMs: p.number,
-
-  // Wait `waitMs` before calling onReady.
-  waitMs: p.number,
-
-  // onSpin is called when the spinner starts spinning.
-  onSpin: p.func,
-
-  // onReady is called once the spinner has spun for `waitMs`.
-  onReady: p.func,
-
-  // text indicates being used as a text placeholder
-  text: p.string,
-}
-
 Spinner.defaultProps = {
   delayMs: DEFAULT_SPIN_DELAY_MS,
   waitMs: DEFAULT_SPIN_WAIT_MS,
```

Comment:
We can destructure the props at the top and use set the default values there instead of having `Spinner.defaultProps`.
```
const { delayMs = DEFAULT_SPIN_DELAY_MS, waitMs = DEFAULT_SPIN_WAIT_MS } = props
```